### DISCUSSION TITLE
Windows issue. no output files generated

### DISCUSSION THREAD

**Discussion by baramousa** [phydrus/phydrus#20] [Compiling Fortran Code] [answered]
Hi I cannot get phydrus to work on windows. i tried it on Linux and it works.

 In windows only the profile, atmosphere in and secelctor are in generated. when i run one of your examples in the cmd i get message that "libquadmath-0.dll" , "libwinpthread-1.dll" and "libgcc-s-seh-1.dll" were not found . I downloaded them but still it wouldn't run. I am susepcting a problem in compiling the source code.  Any suggestions ?

**Comment by raoulcollenteur** [comment, marked as answer]
Did you point the the hydrus.exe file correctly?

**Reply by baramousa** [reply]
yes . i did

**Reply by raoulcollenteur** [reply]
So maybe you need to recompile the fortran code for windows.. did you already do that?

**Reply by baramousa** [reply]
i did so . for windows i needed to download MSYS2 64 in order to run the make file which you provided as well ( because gfortran cannot be run from the commandline in windows), and then i pointed it to the newly created hydrus.exe. Then when i run phydrus it gives the warnings i mentioned above. 

As i said it runs in Linux , but i tried it on two windows computers and i always end up getting the same warnnings .

**Reply by baramousa** [reply]
i finally got it to work on windows. I followed some steps from this youtube video .
https://www.youtube.com/watch?v=wGv2kGl8OV0&list=LL&index=4&t=1s
after downloading MinGW for windows steps are summarized as follows:
-open MinGW manager and click on "all packages". 
-make sure to select the following (mingw developer, mingw make, mingw32-base)
-from "MinGW Base System" select the following (mingw32-base, mingw-gcc, mingw gcc-fortran, mingw-gdb)
-then click on installation and apply changes , after the installation is completed close. 
-from start search and click on "edit the system env.variables"
-click on advanced and then on environment variables. 
-from the poped up window select "path" and click on edit. 
-from the poped up window click new and type the path to bin folder in MinGW. for example ("C:\MinGW\bin"), then ok

after that one should be able to compile the code by opening the command line cmd and moving to the source folder of phydrus and running the "make" file. 

maybe you guys @raoulcollenteur  can add those instruction for people working with windows. it maybe be straight forward for experienced users but i think it is not for beginners like me :) ..

**Reply by raoulcollenteur** [reply]
Thanks @baramousa for the feedback and the detailed explanation on the compilation! I am going to add a section on compiling the fortran code next week and I'll make sure to add this suggestion. Hope you find Phydrus useful for you're work :)

**Reply by InesVid** [reply]
> -from start search and click on "edit the system env.variables" -click on advanced and then on environment variables. -from the poped up window select "path" and click on edit. -from the poped up window click new and type the path to bin folder in MinGW. for example ("C:\MinGW\bin"), then ok 

Hi, I also needed to follow this procedure (nicely written btw!) , but I also needed to include C:\MinGW\msys\1.0\bin to the path variable to get it working.
Cheers,
Ines
